web/components/FocusTrappedSidebar.tsx

```diff
@@ -10,17 +11,21 @@ interface Props {
 }
 
 export const FocusTrappedSidebar = ({ children, close, isOpen, delayTime = 300 }: Props): ReactElement => {
+  const { contextualInfo, setContextualInfo } = useContext(ContextualInfoContext);
   const [showDiv, setShowDiv] = useState(false);
 
   useEffect(() => {
     let timeoutId: ReturnType<typeof setTimeout>;
     if (isOpen && !showDiv) {
       setShowDiv(true);
     } else if (!isOpen && showDiv) {
-      timeoutId = setTimeout(() => setShowDiv(false), delayTime);
+      timeoutId = setTimeout(() => {
+        setShowDiv(false);
+        setContextualInfo({ ...contextualInfo, markdown: "" });
```

Comment:
should this be here? don't we use this component elsewhere, maybe do a callback or something instead?

Reply:
@mikewolfd Thanks for pointing that out. I've made some changes to the code. The markdown property is not being updated within the contextualInfoPanel component.